classes/PodsInit.php

```diff
@@ -1863,6 +1866,38 @@ public function admin_bar_links() {
 
 	}
 
+	/**
+	 * Add Pod fields to user export.
+	 * Requires WordPress 5.4+
+	 *
+	 * @since 2.7.17
+	 *
+	 * @param array    $additional_user_profile_data {
```

Comment:
* :no_entry_sign: Expected 3 spaces after parameter type; 4 found
* :no_entry_sign: Doc comment for parameter $additional_user_profile_data does not match actual variable name $additional_profile_data

<sub>hash: e0fe3c442129825c7c827d2ef58fe3df</sub>